packages/kit/src/runtime/client/client.js

```diff
@@ -33,15 +34,15 @@ const default_error = components[1]();
 // state we're navigating from
 
 /** @typedef {{ x: number, y: number }} ScrollPosition */
-/** @type {Record<number, ScrollPosition>} */
+/** @type {Record<String, ScrollPosition>} */
```

Comment:
We normally use lowercase `string` and `number`

https://stackoverflow.com/a/42654496